.github/workflows/compare-pr-stats.yml

```diff
@@ -2,7 +2,8 @@ name: Compare PR stats
 
 on: [pull_request]
 
-permissions: read-all
+permissions:
+  pull-requests: write
```

Comment:
@tomlongridge these are the same permissions we are currently using on the notifier, the previous configuration was invalid